client/extensions/woocommerce/woocommerce-services/state/shipping-classes/actions.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { errorNotice } from 'state/notices/actions';
+import {
+	WOOCOMMERCE_SERVICES_SHIPPING_CLASSES_REQUEST,
+	WOOCOMMERCE_SERVICES_SHIPPING_CLASSES_REQUEST_SUCCESS,
+} from 'woocommerce/woocommerce-services/state/action-types';
+import { areShippingClassesLoaded, areShippingClassesLoading } from './selectors';
+
+export const fetchShippingClassesSuccess = ( siteId, data ) => {
+	return {
+		type: WOOCOMMERCE_SERVICES_SHIPPING_CLASSES_REQUEST_SUCCESS,
+		siteId,
+		data,
+	};
+};
+
+export const fetchShippingClassesFailure = () => {
+	return errorNotice(
+		`Could not retrieve the shipping classes for this website. Please refresh and try again!`
+	);
+};
+
+export const fetchShippingClassesIfNotLoaded = siteId => ( {
+	type: WOOCOMMERCE_SERVICES_SHIPPING_CLASSES_REQUEST,
+	siteId,
+} );
```

Comment:
Because it made testing much simpler in many instances, as I did not have to always simulate the state, needed for `areShippingClassesLoaded` and `areShippingClassesLoading` in `fetchShippingClasses` (which had it's own test in `test/actions.js`).

Seeing that this is confusing for others, I removed the action and now I'm manually simulating `getState` in all tests.